translator/config/schema.json

```diff
@@ -812,6 +888,11 @@
         "traces_collected": {
           "type": "object",
           "properties": {
+            "app_signals": {
+              "type": "object",
+              "properties": {},
+              "additionalProperties": true
```

Comment:
nit: If this should always be empty, then consider setting `additionalProperties` to false, so the schema validation will fail. This way there's less confusion about the configuration.